react/week4/ResultSearch.js

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from "react";
+import { SearchUserContext } from "./SearchUserContext.js";
+
+export const ResultSearch = () => {
+  const { users, search, err } = useContext(SearchUserContext);
+
+  if (!search) return <p>No results</p>;
+
+  return (
+    <div>
+      {err && <p>{err}</p>}
+      {!err ? (
```

Comment:
Can you please explain the condition here , its not clear by looking into it. So you are checking for error on line number 12 and if there is no error then you are check another condition on line number 14 can't we merge this conditions?